guardian/gudocs
# Update dependencies

## What does this change?

* Update Node version to 16.20.0 
* Update dependencies, most notably `googleapis` to 126.0.1

## How to test

As described in the ReadMe: 
* Run the fetch script to retrieve and parse documents `./dev-start-fetch.sh`
* Run the UI and check that you can publish a document
